cpp/memilio/utils/time_series.h

```diff
@@ -148,6 +178,16 @@ class TimeSeries
     TimeSeries(TimeSeries&& other)            = default;
     TimeSeries& operator=(TimeSeries&& other) = default;
 
+    /// Check if the time is strictly monotonic increasing.
+    bool is_sorted()
```

Comment:
Renamed it.
For interpolation functions (and maybe simulation results as well?) we need strictness. Otherwise, there is a time with two different values, so the function values would not be well defined. And I am not sure how to handle this in general, for example the linear interpolation would have a discontinuity, which I assume we do not want.

I could also add an `is_monotonic`?

Reply:
In principle, simulation results and interpolation can work with discontinuities. In fact, the results we get from graph simulations are discontinuous, this just isn't reflected in the data structure. Since we currently aren't using it, it's OK to go with what you did and don't think about possible extensions.